azure-devops/azext_devops/dev/pipelines/runs.py

```diff
@@ -0,0 +1,67 @@
+# --------------------------------------------------------------------------------------------
```

Comment:
This pipelines folder is for pipelines vertical.
My understanding says that pipeline vertical currently has build, release and pipelines (preview)
Runs as a concept exist across all 3 (build, release, pipelines) so we might need to names these files accordingly to avoid confusion like this should be pipeline_runs. 
* If my understanding about build, release and pipeline being different is correct

Reply:
That is correct pipelines_runs would be a better name for this file.

Reply:
OK - I'll rename to ```pipelines_runs.py``` in the next push. Would be good to get clarity on whether we need to have ```pipelines runs artifacts``` AND ```pipelines build runs artifacts```. To me it seems like unnecesary redundancy.

Reply:
In light of the conversations with the Pipelines team I am proposing that we keen the filename as ```runs.py``` and not rename it to ```pipeline_runs.py``` given that there will be only one ```runs``` command tree. I'm going to resolve this for now since I think that is pretty self evident but if there is another reason to rename to ```pipeline_runs.py``` reopen this comment.